docs/en-US/installation/software-requirements.md

```diff
@@ -25,3 +25,7 @@ VVV requires recent versions of both Vagrant and VirtualBox to be installed, alo
         * If vagrant-triggers is not installed, VVV will not provide automated database backups.
     1. Install the [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) plugin with `vagrant plugin install vagrant-vbguest`.
         * Note: This step is not a requirement. When installed, it keeps the [VirtualBox Guest Additions](https://www.virtualbox.org/manual/ch04.html) kernel modules of your guest synchronized with the version of your host whenever you do `vagrant up`. This can prevent some subtle shared folder errors.
+
+## Windows Users
+
+You may need to turn on virtualisation in your computers BIOS, some computers come with it turned off by default. Refer to your machines manufacturer for how to access your BIOS.
```

Comment:
For better or worse, we already use the US English "virtualization", so we should probably stick with that. 😝

Reply:
Force of habit, British english man using British english :p